tutorials/bigtable-stackdriver/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Using matplotlib to visualize Stackdriver metrics for Bigtable
+description: Learn about using matplotlib to plot Stackdriver metrics for Bigtable
```

Comment:
s/Stackdriver metrics for Bigtable/Stackdriver Monitoring metrics for Cloud Bigtable/